docs/misc/dvd-remux.md

```diff
@@ -0,0 +1,670 @@
+        Once you have registered it,
+        you will need to restart the program.
+
+        !!! example "Selecting titles"
+
+            === "ISO"
+
+                Click the "Open File" button
+                and select your DVDISO.
+                It will start processing the file,
+                and then show you this window:
+
+                ![MakeMKV main window](./img/dvd-remux/makemkv/main-window.png)
+
+                Select the titles you want to remux,
+                give it an output folder and name,
+                and click the button underneath "Make MKV".
+
+                ![MakeMKV Progress window](./img/dvd-remux/makemkv/progress-window.png)
+
+                Once it's done,
+                the files will be in the output folder.
+
+                ![MakeMKV output folder](./img/dvd-remux/makemkv/output.png)
+
+            === "Physical disc"
+
+                Insert your physical disc.
+                MakeMKV will automatically detect it,
+                and then show you this window:
+
+                ![MakeMKV main window](./img/dvd-remux/makemkv/inserted-disc-window.png)
+
+                Click the "Backup" button,
+                select "Decrypt video files"
+                (if possible),
+                pick an output directory,
+                and press "OK".
+                It will now start creating
+                a back-up of your disc.
+
+                ![MakeMKV backup window](./img/dvd-remux/makemkv/copy-dvd-window.png)
+
+                Once it's done,
+                [follow the instructions
+                for the ISO method](#__tabbed_5_1).
+
+## Setting the correct aspect ratio
+
+DVDs (and later SD Blu-rays)
+are stored in a format known as "anamorphic video".
+This means that the stored video dimensions
+differ from the intended display dimensions.
+A typical NTSC DVD stores video at 720x480 pixels
+(a 3:2 aspect ratio),
+but displays it at either widescreen (16:9)
+or standard definition (4:3) aspect ratios.
+
+To achieve the correct display size,
+DVDs make use of a [Sample Aspect Ratio (SAR)](https://en.wikipedia.org/wiki/Sample_aspect_ratio),
+also known as a [Pixel Aspect Ratio (PAR)](https://en.wikipedia.org/wiki/Pixel_aspect_ratio).
+This value tells the player how much to stretch each pixel.
+The system was designed for CRT televisions,
+which had [overscan](https://en.wikipedia.org/wiki/Overscan),
+meaning they would stretch and slightly crop
+the edges of the image,
+which allowed the image to reach its intended [Display Aspect Ratio (DAR)](https://en.wikipedia.org/wiki/Display_aspect_ratio).
+
+At the start of the digital age,
+DVD authors accounted for this
+by keeping important content
+within a smaller [active area](https://en.wikipedia.org/wiki/Overscan#Overscan_amounts).
+When played on a CRT,
+the television's natural stretching and cropping
+would result in the correct final aspect ratio
+being displayed.
```

Comment:
Clarify that it's the *active area* that is displayed with that aspect ratio, rather than the entire stored frame.